internal/policy/policy_test.go

```diff
@@ -94,6 +94,47 @@ func TestLoadCurrentState(t *testing.T) {
 	assert.Equal(t, state, loadedState)
 }
 
+func TestLoadFirstState(t *testing.T) {
+	repo, state := createTestRepository(t, createTestStateWithPolicy)
+
+	// Update policy, record in RSL
+	secondState, err := LoadCurrentState(context.Background(), repo) // secondState := state will modify state as well
+	if err != nil {
+		t.Fatal(err)
+	}
+	targetsMetadata, err := secondState.GetTargetsMetadata(TargetsRoleName)
```

Comment:
```suggestion

	targetsMetadata, err := secondState.GetTargetsMetadata(TargetsRoleName)
```

Reply:
A rule of thumb I somewhat use is to have a blank line after an error check, though sometimes (say when I'm loading multiple keys) it feels okay to put them all together.

Reply:
Fixed

Reply:
I think there are more instances in this block that could be fixed up...

Reply:
I think I have adressed them all?